### PR TITLE
Add service providers

### PR DESCRIPTION
**Pull request by ethervoid**
Related #8989 

Add provider configuration for users and organizations for the geocoder,
isolines and routing services.

### CODE REVIEW

**Comment by ethervoid** [comment]
@jgoizueta please CR this

File: app/models/organization.rb [hunk 1]
```diff
@@ -289,6 +289,9 @@ def to_poro
       :obs_snapshot_block_price => self.obs_snapshot_block_price,
       :obs_general_quota        => self.obs_general_quota,
       :obs_general_block_price  => self.obs_general_block_price,
+      :geocoder_provider => self.geocoder_provider,
```
Comment (jgoizueta) [inline app/models/organization.rb]: should align the `=>`as above (I'm lending a hand to that nice hound)

**Comment by jgoizueta** [comment]
LGTM :+1:

**Comment by juanignaciosl** [comment]
@jgoizueta , @ethervoid , both to avoid errors caused by having code referring to new columns before migration is applied and to allow rollbacks of code without rollbacking migrations, we still deploy migrations in its own PR. A single PR shouldn't have both migrations _and_ code.

**Comment by ethervoid** [comment]
Noted, that's the way I do it every time but any migration of the user table is going to create signup errors for onging user creations. Maybe we should fix that behavior

**Pull request merged**